hack/verify-all.sh

```diff
@@ -19,4 +19,4 @@ set -o pipefail
 
 cd "$(git rev-parse --show-toplevel)"
 find hack -name 'verify-*.sh' -not -name "$(basename "$0")" \( -print -exec '{}' ';' -o -quit \)
-./guberantor/verify_config.sh
+./gubernator/verify_config.sh
```

Comment:
lol, how long has this been broken?

Reply:
not long, I only introduced this within the last N weeks

turns out this won't work anyway...